packages/cli/package.json

```diff
@@ -65,6 +65,8 @@
     "pwa-helpers": "^0.9.1",
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
+    "rehype-autolink-headings": "^4.0.0",
```

Comment:
these don't belong here, do they?  Maybe this was missed when [removing those additional dependencies](https://github.com/ProjectEvergreen/greenwood/pull/374#discussion_r443271611)?

If this is needed for testing, then I think it would be better to move this to the root _package.json_ instead, where are other dev tools are.

Reply:
yes they have to belong there or the tests will not be completed properly

Reply:
can you please move to root _package.json_ then?

Reply:
Oh, I think `devDependencies` would be more accurate

Reply:
yea that's better

Reply:
already got it 😄